### PR TITLE
Feature/capture page reducer

### PR DESCRIPTION
**Pull request by hui-mu**
Main changes on the capture page
1. Create the appSlice for the client's application
2. Writing fake requests in the backend for testing
3. Implement functionality of start button, stop button
4. Implement polling for app process and capturing process
5. Fix typescript and eslint issue

Potential issues
1. logs
2. comments
They should be moved after intergration with the backend
3. writing test file for manage page component

### CODE REVIEW

**Review by 8000meters — CHANGES_REQUESTED** [review]
Did you pull from dev and merge before making this PR? it doesn't have the new code from the back-end.

File: src/public/infernode/views/Capture/CapturePage.tsx [hunk 1]
```diff
@@ -9,80 +11,171 @@ import {
...
+              <Form.Group className="mb-3" controlId="formGroupPath">
+                <Form.Label>Relative Path of Entry Point</Form.Label>
+                <Form.Control type="text" id="relativePath" placeholder="/enter/relative/path" />
+              </Form.Group>
+            </Form>
+            <div className="d-grid gap-2">
+              <Button
+                variant="success"
+                size="lg"
+                onClick={() => {
```
Comment (seadubs) [inline src/public/infernode/views/Capture/CapturePage.tsx:83]: What are your thoughts on refactoring this out of an inline function into a named function declared outside of the return scope? Given that it's more than a one-liner, it makes it a bit hard to read the overall module having functionality in multiple places.

Comment (hui-mu) [reply src/public/infernode/views/Capture/CapturePage.tsx:83]: Good suggestions! Thank you!

**Review by dzidupe — DISMISSED** [review]
Looks good!

**Review by 8000meters — APPROVED** [review]
looks good. thanks for updating

**Pull request merged**